warehouse/models/staging/transit_database/base/_base_transit_database.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 models:
-  - name: int_tts_services_ct_services_map
+  - name: base_tts_services_ct_services_map
```

Comment:
I don't think you can use multiple combination_of_columns keys under the same unique_combination_of_columns test; the YAML spec is to ignore dupes IIRC